README.md

```diff
@@ -1,12 +1,44 @@
+RVM uses GitHub Actions to automate the role creation and deployment process. There are two main types of roles that you can create and manage using the Role Vending Machine:
+
+- **Machine Roles**: These are roles that are intended to be used by AWS services or GitHub Actions to access resources on behalf of the service or application. They provide the necessary permissions for the service to perform its intended functions.
+- **Break Glass Access Roles**: These are special roles that provide emergency or temporary access to AWS accounts, for example if your main IdP provider is experiencing an outage and you need console access to your AWS accounts.
+
+## Machine roles
+
+RVM allows developers to create roles for three different types of principals: GitHub pipelines, EKS pods, and other AWS services. Developers will commit Terraform files outlining required permissions and other essential details for their workload to RVM repository. When they create a pull request, a GitHub workflow initiates a Terraform plan to summarize the deployment changes, scans the Terraform template using Checkov, validates the submitted IAM policies against [IAM Access Analyzer policy check reference](https://docs.aws.amazon.com/IAM/latest/UserGuide/access-analyzer-reference-policy-checks.html), and adds this information to the pull request. Following a review and approval, and after merging the pull request, another workflow executes the Terraform apply command to deploy the proposed role in the target AWS account.
 
 ![RVM Workflow](assets/rvm-workflow.png)
 
+## Break Glass Access Roles
+
+Break glass access refers to a quick, emergency means for a person who does not normally have access privileges to certain AWS accounts to gain access in exceptional circumstances. This is done by using an approved process, similar to breaking the glass to trigger a fire alarm. The key use cases for break glass access include:
+
+- Failure of the organization's identity provider (IdP)
+- Security incidents involving the organization's IdP(s)
+- Failures with IAM Identity Center
+- Disasters resulting in the loss of the organization's cloud or identity management teams
+
+![RVM Break Glass](assets/breakglass.png)
+
+The Role Vending Machine takes a unique approach to managing break glass access, rather than creating dedicated break glass user accounts and roles in each AWS account. Instead, it:
+
+- Creates a temporary role with the requested permissions when break glass access is needed, deploying it directly in the target account.
```

Comment:
Is this role actually temporary? Or is it just the credentials that are temporary?